llvm/test/CodeGen/SystemZ/fp-abs-01.ll

```diff
@@ -3,6 +3,17 @@
 ; RUN: llc < %s -mtriple=s390x-linux-gnu -mcpu=z10 | FileCheck %s
 ; RUN: llc < %s -mtriple=s390x-linux-gnu -mcpu=z13 | FileCheck %s
 
+; Test f16.
+declare half @llvm.fabs.f16(half %f)
+define half @f0(half %f) {
+; CHECK-LABEL: f0:
+; CHECK: brasl %r14, __extendhfsf2@PLT
+; CHECK: lpdfr %f0, %f0
```

Comment:
There should also be a truncsfhf2 here, right?   Probably best to use a CHECK-NEXT sequence.